Documentation/CRDs/Cluster/external-cluster.md

```diff
@@ -45,7 +45,7 @@ python3 create-external-cluster-resources.py --rbd-data-pool-name <pool_name> --
 - `--skip-monitoring-endpoint`: (optional) Skip prometheus exporter endpoints, even if they are available. Useful if the prometheus module is not enabled
 - `--ceph-conf`: (optional) Provide a Ceph conf file
 - `--keyring`: (optional) Path to Ceph keyring file, to be used with `--ceph-conf`
-- `--cluster-name`: (optional) Ceph cluster name
+- `--k8s-cluster-name`: (optional) Ceph cluster name
```

Comment:
```suggestion
- `--k8s-cluster-name`: (optional) Kubernetes cluster name
```
it should be the kubernetes cluster name or ceph cluster name? the key and the description should match

Reply:
Updated